posthog/client.py

```diff
@@ -241,21 +241,21 @@ def group_identify(self, group_type=None, group_key=None, properties=None, conte
 
         return self._enqueue(msg)
 
-    def alias(self, previous_id=None, distinct_id=None, context=None, timestamp=None, uuid=None):
+    def alias(self, distinct_id=None, previous_id=None, context=None, timestamp=None, uuid=None):
         context = context or {}
 
         require("previous_id", previous_id, ID_TYPES)
         require("distinct_id", distinct_id, ID_TYPES)
 
         msg = {
             "properties": {
-                "distinct_id": previous_id,
-                "alias": distinct_id,
+                "distinct_id": distinct_id,
+                "alias": previous_id,
             },
             "timestamp": timestamp,
             "context": context,
             "event": "$create_alias",
-            "distinct_id": previous_id,
+            "distinct_id": distinct_id,
```

Comment:
why are we swapping the order here?

Reply:
In our talks we tell people to use the ordering posthog.alias(distinct_id, alias). However, the parameter names are the wrong way round for this so I've updated the parameter names. Even in the current version we're actually sending it the right way (but in a fairly hacky way where we send the distinct_id as the alias because the user actually gives the alias in place of the distinct_id)